app/models/incident_report.py

```diff
@@ -11,6 +11,10 @@ class Location(db.Model):
     incident_report_id = db.Column(db.Integer,
                                    db.ForeignKey('incident_reports.id'))
 
+    def __repr__(self):
+        # TODO: Show address instead?
```

Comment:
Location used to be one of the columns for viewing all reports, but I took it out. It's currently not used anywhere, but I think it's a helpful function to have so I left it in.

What do you mean by showing everything? What else can we show for location besides lat and long?